itou/www/apply/views/list_views.py

```diff
@@ -44,17 +49,25 @@ def list_for_prescriber(request, template_name="apply/list_for_prescriber.html")
     job_applications = get_all_available_job_applications_as_prescriber(request)
 
     filters_form = PrescriberFilterJobApplicationsForm(job_applications, request.GET or None)
-    filters = None
 
-    job_applications = job_applications.with_list_related_data()
+    # Add related data giving the criteria for adding the necessary annotations
+    job_applications = job_applications.not_archived().with_list_related_data(
+        filters_form.data.getlist("criteria", [])
```

Comment:
```suggestion
        criteria=filters_form.data.getlist("criteria", [])
```
On ne sait jamais, on peut être amenés à ajouter des paramètres dans le futur.